tests/system/large/ml/test_linear_model.py

```diff
@@ -31,8 +31,14 @@ def test_linear_regression_configure_fit_score(penguins_df_default_index, datase
         ]
     ]
     y_train = df[["body_mass_g"]]
+
+    start_execution_count = df._block._expr.session._metrics.execution_count
+
     model.fit(X_train, y_train)
```

Comment:
I think we have the opportunity to make this work work all ML APIs - fit, score, predict, transform, .... We should write such tests for all APIs by writing the metrics collection logic in more central place in the code.

Reply:
score, predict works, seems internally is calling other functions.

Reply:
Good to know, we could just pick one or two of the existing tests that does fit, score, predict, register and transform and assert everywhere that we are counting stats for those operations.

Reply:
Test Added.